src/components/Home/Home.js

```diff
@@ -39,7 +39,11 @@ const Home = ({ history }) => {
       setFetching(false);
     }).catch((err) => {
       if (err.status === 401) {
-        history.push('/login');
+        const redirect = {
```

Comment:
If you agree on refactoring here: https://github.com/codurance/Retropolis-FE/pull/47/files#r436666191 please reuse that function you extracted.